OrbitCore/CaptureData.h

```diff
@@ -118,6 +142,10 @@ class CaptureData {
 
   absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionStats>
       functions_stats_;
+
+  std::unique_ptr<CallstackData> callstack_data_;
```

Comment:
Any reason for those two the be unique_ptrs rather than just members? Just a clarification question.

Reply:
As discussed offline, just add a comment with the explanation here.

Reply:
Done